WordPress/Classes/Extensions/Media+Blog.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+extension Media {
+
+    /// Inserts and returns a new managed Media object, in the context.
+    ///
+    class func inserted(into context: NSManagedObjectContext) -> Media {
```

Comment:
I _think_ `inserted` fits better with Swift 3 naming because it returns a new object (that has been _inserted_). If it was an instance method that was inserting an existing object then I think `insert` would fit better.